openapi.yaml

```diff
@@ -0,0 +1,121 @@
+openapi: 3.0.3
+info:
+  title: Signal K Sails configuration
+  description: API for managing the state of the vessel's sails with Signal K
+  version: 0.1.0
+servers:
+  - url: http://localhost:3000
+tags:
+  - name: sails
+    description: Sail management
+paths:
+  /sails:
+    get:
+      tags:
+      - sails
+      summary: Get current sails state
+      description: Read the current state of sails on the vessel
+      responses:
+        '200':
+          description: Current state
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/sails'
+    put:
+      tags:
+      - sails
+      summary: Set current sails state
+      description: Update the current state of sails on the vessel
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/sails'
+      responses:
+        '200':
+          description: Successful operation
+  /set/{id}:
```

Comment:
Instead of "verb" like, "imperative" paths like `set` and `reef` followed by sail id I think we could leverage http PUT to directly modify individual properties in the data model:
- PUT `/sails/{id}/active` payload `true` | `false`
- PUT `/sails/{id}/reducedState` payload `{"reefs":2}`  |  `{"furledRatio": 0.8}`

Reply:
Express body parser in Signal K is set to "strict mode" so it only accepts objects and arrays. So the payload for `active` endpoint will be `{ value: BOOL }`